src/components/AddItem.js

```diff
@@ -3,10 +3,12 @@ import { writeToFirestore } from '../lib/firebase';
 
 const AddItem = () => {
   let [name, setName] = useState('');
+  const token = localStorage.getItem('token');
+
   function handleSubmitForm(event) {
     event.preventDefault();
     if (name.length > 0) {
-      writeToFirestore('shopping-list', { name });
+      writeToFirestore(token, { name });
```

Comment:
same

Reply:
@laurenyz @TianaJohnson 2/2 instances of updating the collection name to the user's token. Again, let @QMaximillian or I know if there are any issues with this that we need to retool.